attack/scripts/bashrc

```diff
@@ -61,6 +61,44 @@ info() {
 } 1>&2
 readonly -f info
 
+#
+# Starting point.
+#
+
+starting_point() {
+  # Identify the starting point mode.
+  MODE="$(sed '3,3!d' /tasks.yaml)"
+  export MODE
+
+  #Determine the type.
+  if [[ $MODE =~ "internal-instance" ]]; then
+    ssh_internal
+  else
+    if [[ $MODE =~ "node" ]]; then
+      NODE_TYPE="$(sed '4,4!d' /tasks.yaml)"
+      export NODE_TYPE
+
+      if [[ $NODE_TYPE =~ "master" ]]; then
+        NODE_NUMBER="$(echo $NODE_TYPE | sed "s/node-id: 'master-//" | sed "s/'//")"
+        export NODE_NUMBER
+        ssh_master $NODE_NUMBER
+      else
+        NODE_NUMBER="$(echo $NODE_TYPE | sed "s/node-id: 'node-//" | sed "s/'//")"
+        export NODE_NUMBER
+        ssh_node $NODE_NUMBER
+      fi
+    else
+      if [[ $MODE =~ "pod" ]]; then
+        echo "Starting point: POD - PENDING IMPLEMENTATION"
+      else
+        echo "There's no starting point or the scenario configuration file is not following the expected format."
```

Comment:
we should disambiguate these 2 conditions (no starting point vs bad format)  .. using yq could help here

Reply:
Covered in the new commit.